xml/System.Net.PeerToPeer.Collaboration/PeerObject.xml

```diff
@@ -279,7 +279,7 @@
       <Docs>
         <param name="other">The <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance to test for equality.</param>
         <summary>Compares a <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> to the specified <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance.</summary>
-        <returns>Returns <see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance provided has matching data, else <see langword="false" />.</returns>
+        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance provided has matching data, else <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance provided has matching data; otherwise, <see langword="false" />.</returns>
```